scripts/license.mjs

```diff
@@ -67,16 +67,17 @@ const files = await glob(
       '**/LICENSE.md',
       '**/CHANGELOG.md',
       '**/.github/**/*.md',
-      '**/*.yml',
     ],
   },
 );
 
 for (const file of files) {
-  const header = getFormattedHeader(path.extname(file));
+  const fileExtension = path.extname(file);
+  const isFileExtensionSupported = fileExtension in COMMENT_STYLES;
 
   let contents = fs.readFileSync(file, 'utf8');
-  if (!/Copyright \d+ Hypergiant/.test(contents)) {
+  if (isFileExtensionSupported && !/Copyright \d+ Hypergiant/.test(contents)) {
```

Comment:
Perhaps a small note indicating that the `isFileExtensionSupported` primarily supports the use case of the passed in files vs. the predefined glob we have in this file